keps/prod-readiness/sig-network/4974.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group 
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 4974
+deprecated:
+  approver: "soltysh"
```

Comment:
Discussed this and @wojtek-t will be taking this one over.

Reply:
+1 - please switch to me.